OtripleS.Web.Api/Brokers/Storage/IStorageBroker.Courses.cs

```diff
@@ -3,17 +3,19 @@
 // FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
 // ---------------------------------------------------------------
 
-using Microsoft.EntityFrameworkCore;
-using OtripleS.Web.Api.Models.Course;
 using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using OtripleS.Web.Api.Models.Course;
```

Comment:
@viralpandya  I must have missed this in @HatemGamal  PR.
Please change this to say `Courses` the same thing goes with the folder names for both models and brokers.

Reply:
Should update in the evening

Reply:
its updated. Please check and let me know, if more changes are needed.